ironfish/src/rpc/routes/chain/getBlock.ts

```diff
@@ -124,6 +128,10 @@ router.register<typeof GetBlockRequestSchema, GetBlockResponse>(
       throw new ValidationError(error)
     }
 
+    if (header.noteSize === null) {
```

Comment:
Adding this here because we [never save to database without the noteSize](https://github.com/iron-fish/ironfish/blob/8b765694d9350e19b82ff6886457f9ff992b7fef/ironfish/src/blockchain/database/headers.ts#L18). Ideal solution would be to have separate types for a network block header and a database block header but we don't have that yet. This should never be null so failing loudly here